### DISCUSSION TITLE
How to define a rigidbody using imported geometry?

### DISCUSSION THREAD

**Discussion by qiushi-Lee** [jgerstmayr/EXUDYN#36] [Q&A on usage of Exudyn] [answered]
I'm sorry to bother you again. I see your examples of multi-rigidbody in exudyn. The VObjectRigidBody are all built by netgen or embedded geometry.  I'd like to know whether the VObjectRigidBody could use the imported geometry (STP or the others format)as the graphics. Thanks.

**Comment by jgerstmayr** [comment, marked as answer]
Rigid bodies can be visualized by triangular geometries. You can import STL meshes, see module graphicsDataUtilities: 
functions GraphicsDataFromSTLfileTxt and GraphicsDataFromSTLfile

[Check the new online documentation](https://exudyn.readthedocs.io/)
